src/Passwordless/Models/Credential.cs

```diff
@@ -22,18 +29,76 @@ public class Credential
         UserId = userId;
     }
 
+    /// <summary>
+    /// Descriptor of the credential as defined by the WebAuthn specification
+    /// <see href="https://w3c.github.io/webauthn/#enumdef-publickeycredentialtype" />
+    /// </summary>
     public CredentialDescriptor Descriptor { get; }
+
+    /// <summary>
+    /// Public key of the passkey pair.
+    /// </summary>
     public byte[] PublicKey { get; }
+
+    /// <summary>
+    /// Byte array of user identifier
+    /// </summary>
     public byte[] UserHandle { get; }
+
+    /// <summary>
+    /// Number of times credential has been used
```

Comment:
```suggestion
    /// WebAuthn SignatureCounter, used for anti forgery. 
```

Often you're correct, but Apple and some others use set unix time to this counter.